simulator/simulator/rusEfiFunctionalTest.cpp

```diff
@@ -84,7 +84,7 @@ static void runChprintfTest() {
 
 }
 
-void rusEfiFunctionalTest(void) {
+void rusEfiFunctionalTest(char const * const socketcanDevice) {
 	printToConsole("Running rusEfi simulator version:");
```

Comment:
wish there was a `printfToConsole` ;)

Reply:
you mean to the FOME console? that exists, it's called `efiPrintf`